File: Selenium/Project/HRM_Activity6.java
package Project;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.WebDriverWait;
import org.testng.Assert;

import java.time.Duration;


public class HRM_Activity6 {

    public static void main(String[] args) throws InterruptedException {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(5));
        driver.get("http://alchemy.hguy.co/orangehrm");
        driver.findElement(By.id("txtUsername")).sendKeys("orange");
        driver.findElement(By.id("txtPassword")).sendKeys("orangepassword123");
        driver.findElement(By.id("btnLogin")).click();
        Thread.sleep(2000);

        // Verify Directory
        WebElement menuitem=driver.findElement(By.xpath("//div/ul/li[9]/a/b"));
        if(menuitem.isDisplayed())
        {
            menuitem.click();
        }

        //Verify header
        WebElement header= driver.findElement(By.xpath("//div[@class='head']/h1"));
        Assert.assertEquals(header.getText(),"Search Directory");

        driver.close();


    }
}

File: Selenium/Project/HRM_Activity6.py
import time

from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from selenium.webdriver.support.select import Select
from selenium.webdriver.support import expected_conditions
from selenium.webdriver.support.wait import WebDriverWait
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service) as driver:

    wait = WebDriverWait(driver, 10)
    driver.get("http://alchemy.hguy.co/orangehrm")
    driver.find_element(By.ID,"txtUsername").send_keys("orange")
    driver.find_element(By.ID,"txtPassword").send_keys("orangepassword123")
    driver.find_element(By.ID,"btnLogin").click()
    time.sleep(2)

    #Verify Directory
    menuitem=driver.find_element(By.XPATH,"//div/ul/li[9]/a/b")
    if(menuitem.is_displayed()):
            menuitem.click()

    #Verify header
    header= driver.find_element(By.XPATH,"//div[@class='head']/h1")
    if(header.text=="Search Directory"):
        print("Text matches")
    else:
        print("Text doesn't match")

    driver.close()
